app/subscriber/src/components/navbar/Navbar.tsx

```diff
@@ -27,12 +32,21 @@ export const Navbar: React.FC<INavbarProps> = ({ options }) => {
   const groupedOptions = options?.filter((option) => !!option.groupName);
   const groupByName = _.groupBy(groupedOptions, 'groupName');
 
+  const determineClassName = React.useCallback((path: string) => {
+    if (window.location.pathname.includes(path)) return 'active';
```

Comment:
React will not know when `window.location.pathname` changes natively.  I believe you need to use the `useLocation()`.

Reply:
updated!